docs/book/src/tasks/experimental-features/cluster-class/write-clusterclass.md

```diff
@@ -587,6 +587,9 @@ constant default value which can be specified in the schema is not enough.
           # If .vnetName is set, it is used. Otherwise, we will use `{{.builtin.cluster.name}}-vnet`.  
           template: "{{ if .vnetName }}{{.vnetName}}{{else}}{{.builtin.cluster.name}}-vnet{{end}}"
 ```
+When writing templates, a subset of functions from [sprig](http://masterminds.github.io/sprig/) can be used to
+write expressing like e.g. `{{ .name | upper }}`. Only functions that are guaranteed to evaluate to the same result
+for a given input are allowed (e.g. `upper` or `max` can be used, while `now` or `randAlpha` can not be used).
 
```

Comment:
Can we document all the available functions in here?

Reply:
Can we instead refer to e.g. that documentation page: http://masterminds.github.io/sprig/

I would really prefer not duplicating part of the docs (even if it's only a list of funcs).

Reply:
there are over 70 template functions in sprig, I don't this it is doable to document here.

Reply:
Linking or importing works, from a user experience having them in one place (even if generate) is much more useful when implementing a clusterclass